admin_tools/menu/items.py

```diff
@@ -189,7 +189,7 @@ def init_with_context(self, context):
         items = self._visible_models(context['request'])
         apps = {}
         for model, perms in items:
-            if not (perms['change'] or perms['view']):
+            if not (perms['change'] or perms.get('view', False)):
```

Comment:
I think this is wrong, IMO it should be:
`if not perms['change'] and not perms.get('view', False)`

Or maybe a better solution would be to do like in dashboard/modules.py:
`if perms['change'] or perms.get('view', False)`

Reply:
Nevermind, I misread the if, your condition is correct !